runtime/src/bank.rs

```diff
@@ -795,12 +789,11 @@ impl Bank {
         // TODO: put this assert back in
         // assert!(!self.is_frozen());
         inc_new_counter_debug!("bank-register_tick-registered", 1);
-        let current_tick_height = self.tick_height.fetch_add(1, Ordering::Relaxed) as u64;
+        let mut blockhash_queue = self.blockhash_queue.write().unwrap();
```

Comment:
Main concern here is that as we scale ticks per slot, this will conflict more with banking stage age checks and RPC client requests for latest blockhash which grab the blockhash queue read lock.

Reply:
yeah, good point